_security/authentication-backends/jwt.md

```diff
@@ -0,0 +1,295 @@
+---
+layout: default
+title: JSON Web Token
+parent: Authentication backends
+nav_order: 47
+redirect_from:
+---
+
+
+# JSON Web Token
+
+JSON Web Tokens (JWTs) are JSON-based access tokens that assert one or more claims. They are commonly used to implement single sign-on (SSO) solutions and fall in the category of token-based authentication systems:
```

Comment:
Can you introduce this process better? Something like, "This is a typical authentication journey using JWT:"

Reply:
did it. Thanks.

Reply:
@hdhalter Can you approve this so it's ready for editorial review? Thanks.